### ISSUE TITLE
Traefik for Windows crash after CTRL+C

### ISSUE THREAD

**Issue by sosoba** [traefik/traefik#7425]
### Do you want to request a *feature* or report a *bug*?

Bug

### What did you do?

Run `traefik.exe` via `cmd.exe` console and press `CTRL+C`.

### What did you expect to see?

Cursor prompt. Else "Program terminated" log.

### What did you see instead?

Error stack

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
2.3.1 windows amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```toml
# empty configuration
```

Windows 10 Pro 19041.508

### If applicable, please paste the log output in DEBUG level (`--log.level=DEBUG` switch)

```
C:\Programs>traefik.exe
runtime: bad pointer in frame github.com/konsorten/go-windows-terminal-sequences.EnableVirtualTerminalProcessing at 0xc00054d680: 0x58
fatal error: invalid pointer found on stack

runtime stack:
runtime.throw(0x378e6a0, 0x1e)
        /usr/local/go/src/runtime/panic.go:1116 +0x79 fp=0x9cabbff0b8 sp=0x9cabbff088 pc=0x69e979
runtime.adjustpointers(0xc00054d680, 0x9cabbff1a8, 0x9cabbff538, 0x40c7db0, 0x5392ea0)
        /usr/local/go/src/runtime/stack.go:605 +0x1e8 fp=0x9cabbff108 sp=0x9cabbff0b8 pc=0x6b5628
runtime.adjustframe(0x9cabbff448, 0x9cabbff538, 0x5392ea0)
        /usr/local/go/src/runtime/stack.go:647 +0x36b fp=0x9cabbff1d8 sp=0x9cabbff108 pc=0x6b59ab
runtime.gentraceback(0xffffffffffffffff, 0xffffffffffffffff, 0x0, 0xc000644c00, 0x0, 0x0, 0x7fffffff, 0x3887e18, 0x9cabbff538, 0x0, ...)
        /usr/local/go/src/runtime/traceback.go:334 +0x11b5 fp=0x9cabbff4b0 sp=0x9cabbff1d8 pc=0x6c36d5
runtime.copystack(0xc000644c00, 0x4000)
        /usr/local/go/src/runtime/stack.go:903 +0x2b9 fp=0x9cabbff668 sp=0x9cabbff4b0 pc=0x6b6139
runtime.newstack()
        /usr/local/go/src/runtime/stack.go:1069 +0x2a5 fp=0x9cabbff818 sp=0x9cabbff668 pc=0x6b64c5
runtime.morestack()
        /usr/local/go/src/runtime/asm_amd64.s:449 +0x97 fp=0x9cabbff820 sp=0x9cabbff818 pc=0x6d4a57

goroutine 74 [copystack]:
runtime.concatstring2(0xc00054d3d8, 0x374c808, 0xc, 0x3735973, 0x1, 0x203, 0x0)
        /usr/local/go/src/runtime/string.go:58 +0x71 fp=0xc00054d3a0 sp=0xc00054d398 pc=0x6b7f31
syscall.UTF16FromString(0x374c808, 0xc, 0x0, 0x0, 0x0, 0x0, 0x0)
        /usr/local/go/src/syscall/syscall_windows.go:46 +0xcc fp=0xc00054d488 sp=0xc00054d3a0 pc=0x73dfcc
syscall.UTF16PtrFromString(...)
        /usr/local/go/src/syscall/syscall_windows.go:96
syscall.LoadDLL(0x374c808, 0xc, 0xc000613588, 0x664a71, 0x671370)
        /usr/local/go/src/syscall/dll_windows.go:70 +0x54 fp=0xc00054d540 sp=0xc00054d488 pc=0x738334
syscall.(*LazyDLL).Load(0xc000004620, 0x0, 0x0)
        /usr/local/go/src/syscall/dll_windows.go:245 +0xc8 fp=0xc00054d590 sp=0xc00054d540 pc=0x73a1c8
syscall.(*LazyProc).Find(0xc000120bd0, 0x0, 0x0)
        /usr/local/go/src/syscall/dll_windows.go:300 +0xbf fp=0xc00054d5e8 sp=0xc00054d590 pc=0x73a4df
syscall.(*LazyProc).mustFind(...)
        /usr/local/go/src/syscall/dll_windows.go:318
syscall.(*LazyProc).Call(0xc000120bd0, 0xc00053c040, 0x2, 0x2, 0xc0006136b8, 0x7469f7, 0x7ffe74d35290, 0x2)
        /usr/local/go/src/syscall/dll_windows.go:336 +0x36 fp=0xc00054d638 sp=0xc00054d5e8 pc=0x73a6f6
github.com/konsorten/go-windows-terminal-sequences.EnableVirtualTerminalProcessing(0x58, 0x1, 0xc0001011d0, 0xc0006136ec)
        /go/pkg/mod/github.com/konsorten/go-windows-terminal-sequences@v1.0.2/sequences.go:30 +0xcb fp=0xc00054d698 sp=0xc00054d638 pc=0xa2e1ab
github.com/sirupsen/logrus.initTerminal(0x3c22680, 0xc000006018)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/terminal_check_windows.go:16 +0x65 fp=0xc00054d6c8 sp=0xc00054d698 pc=0xa353c5
github.com/sirupsen/logrus.checkIfTerminal(0x3c22680, 0xc000006018, 0x26f580f0e58)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/terminal_check_windows.go:31 +0xae fp=0xc00054d700 sp=0xc00054d6c8 pc=0xa3548e
github.com/sirupsen/logrus.(*TextFormatter).init(...)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/text_formatter.go:86
github.com/sirupsen/logrus.(*TextFormatter).Format.func1()
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/text_formatter.go:170 +0x58 fp=0xc00054d730 sp=0xc00054d700 pc=0xa38cd8
sync.(*Once).doSlow(0xc0002162c0, 0xc000613960)
        /usr/local/go/src/sync/once.go:66 +0xf7 fp=0xc00054d780 sp=0xc00054d730 pc=0x6fcdd7
sync.(*Once).Do(...)
        /usr/local/go/src/sync/once.go:57
github.com/sirupsen/logrus.(*TextFormatter).Format(0xc000216280, 0xc0001ae0e0, 0xc000613d38, 0xa38b74, 0x336f4a0, 0xc000544000, 0xc000613d80)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/text_formatter.go:170 +0xf78 fp=0xc00054dd08 sp=0xc00054d780 pc=0xa365d8
github.com/sirupsen/logrus.(*Entry).write(0xc0001ae0e0)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:255 +0x83 fp=0xc00054dd90 sp=0xc00054dd08 pc=0xa2fb83
github.com/sirupsen/logrus.Entry.log(0xc000168000, 0xc000616060, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:231 +0x1af fp=0xc00054de08 sp=0xc00054dd90 pc=0xa2f7ef
github.com/sirupsen/logrus.(*Entry).Log(0xc0001c0460, 0x2, 0xc00064a030, 0x1, 0x1)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:268 +0xf2 fp=0xc00054df10 sp=0xc00054de08 pc=0xa2fe12
github.com/sirupsen/logrus.(*Entry).Error(0xc0001c0460, 0xc00064a030, 0x1, 0x1)
        /go/pkg/mod/github.com/sirupsen/logrus@v1.4.2/entry.go:297 +0x58 fp=0xc00054df48 sp=0xc00054df10 pc=0xa30198
github.com/traefik/traefik/v2/pkg/server.(*TCPEntryPoint).Start(0xc00020cfc0, 0x3c93b00, 0xc0006160c0)
        /go/src/github.com/traefik/traefik/pkg/server/server_entrypoint_tcp.go:174 +0x175 fp=0xc00054dfc8 sp=0xc00054df48 pc=0x2cc6fb5
runtime.goexit()
        /usr/local/go/src/runtime/asm_amd64.s:1374 +0x1 fp=0xc00054dfd0 sp=0xc00054dfc8 pc=0x6d66a1
created by github.com/traefik/traefik/v2/pkg/server.TCPEntryPoints.Start
        /go/src/github.com/traefik/traefik/pkg/server/server_entrypoint_tcp.go:87 +0x147
```

**Comment by jspdown** [comment]
Hello @sosoba and thanks for your interest in Traefik.

I looked at the issue and this panic is linked to a dependency in logrus (our logger): https://github.com/sirupsen/logrus/issues/1112
Since then, the issue has been fixed on https://github.com/konsorten/go-windows-terminal-sequences/commit/71a52a12b06309f8b640fb3169d40575b2085a98 (v1.0.3). Logrus has updated `konsorten/go-windows-terminal-sequences` to v1.0.3 in v1.6.0: https://github.com/sirupsen/logrus/commit/e79215d3d55027d19fd4b4ad8818caf586eafdd7.

As we are currently using logrus v1.4.1 updating it to v1.6.0 should fix this issue.
